Runtimes/Core/CMakeLists.txt

```diff
@@ -103,6 +103,13 @@ option(SwiftCore_ENABLE_FILESYSTEM_SUPPORT "Build for systems that have a filesy
 option(SwiftCore_ENABLE_OS_TRACE_LAZY_INIT "Use os_trace call to check if lazy init has been completed before making os_signpost calls." ${SwiftCore_HAS_OS_TRACE})
 option(SwiftCore_HAS_DARWIN_LIBMALLOC "Use Darwin malloc features" ${APPLE})
 
+if("${APPLE}" OR "${LINUX}")
```

Comment:
Just realized now I can rewrite this with generator expressions

Reply:
As per [this comment](https://github.com/swiftlang/swift/pull/79188#discussion_r1945139343), I cannot use generator expressions to set default values for options.